classes/class-wc-connect-functions.php

```diff
@@ -19,5 +19,25 @@ public static function should_send_cart_api_request() {
 				( isset( $_GET['rest_route'] ) && isset( $_GET['_for'] ) && ( 'jetpack' === $_GET['_for'] ) )
 			);
 		}
+
+		/**
+		 * Get the WC Helper authorization information to use with WC Connect Server requests( e.g. site ID, access token).
+		 *
+		 * @return array|WP_Error
+		 */
+		public static function get_wc_helper_auth_info() {
+			if ( class_exists( 'WC_Helper_Options' ) && is_callable( 'WC_Helper_Options::get' ) ) {
+				$helper_auth_data = WC_Helper_Options::get( 'auth' );
+			}
+
+			// It's possible for WC_Helper_Options::get() to return false, throw error if this is the case.
+			if ( ! $helper_auth_data ) {
+				return new WP_Error(
+					'missing_wccom_auth',
+					__( 'Unable to send request to WooCommerce Shipping & Tax server. WooCommerce Helper auth is missing', 'woocommerce-services' )
```

Comment:
If we are still sending requests even without the Helper auth, this error message could probably be trimmed down to just call out the missing Helper auth.

Reply:
👍